view/frontend/layout/default_head_blocks.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../../../vendor/magento/framework/Module/etc/module.xsd">

```

Comment:
Can you please change this to `xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd"`